bin/.travis/check_code_style.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ -z "${TRAVIS_PULL_REQUEST}" || -z "${TRAVIS_PULL_REQUEST_BRANCH}" ]]
+then
+    echo "> Code Style check: missing required Travis environment variables TRAVIS_PULL_REQUEST or TRAVIS_PULL_REQUEST_BRANCH" >&2;
+    exit 1;
+fi
+
+if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]
+then
+    # limit checking to the files which were affected by a current PR
+    FILES_LIST=$(git diff --diff-filter=ACMR --name-only "${TRAVIS_PULL_REQUEST_BRANCH}" "*.php"|paste -sd ' ');
+    if [[ -z "${FILES_LIST}" ]]
+    then
+        echo "> Code Style check: nothing to check"
+        exit 0;
+    fi
+    echo "> Code Style check: checking the following files: ${FILES_LIST}";
+else
+    # for non-PR builds check entire codebase
+    FILES_LIST=""
+    echo "> Code Style check: checking the entire codebase";
```

Comment:
Note: Makes sense for me to do it on merges and pushes as occasionally we end up with such issues where it was missed on a PR.